pychromecast/__init__.py

```diff
@@ -171,6 +171,33 @@ def get_chromecast_from_uuid(uuid: UUID) -> Chromecast:
     return (list(cc_list.values()), browser)
 
 
+@overload
+def get_chromecasts(
+    tries: int | None = None,
+    retry_wait: float | None = None,
+    timeout: float | None = None,
+    blocking: Literal[True] = True,
+    callback: Callable[[Chromecast], None] | None = None,
+    zeroconf_instance: zeroconf.Zeroconf | None = None,
+    known_hosts: list[str] | None = None,
+) -> tuple[list[Chromecast], CastBrowser]:
+    ...
+
+
+@overload
+def get_chromecasts(
+    tries: int | None = None,
+    retry_wait: float | None = None,
+    timeout: float | None = None,
+    *,
```

Comment:
This is added to allow specifying the value of blocking, which is otherwise not possible.
More details here: https://github.com/python/mypy/issues/7333